src/test/components/TabSelectorMenu.test.js

```diff
@@ -48,4 +66,87 @@ describe('app/TabSelectorMenu', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('should switch tabs properly', () => {
+    const { getState, getByText, firstTabTabID, secondTabTabID } = setup();
+
+    // Check that there is no tab filter at first.
+    expect(getTabFilter(getState())).toBe(null);
+
+    // Change the tab filter by clicking on the menu item.
+    const mozillaTab = getByText('mozilla.org');
```

Comment:
once you'll move to a button, you can use `screen.getByRole('button', { name: 'mozilla.org' })`, this also checks for the accessibility of the element (does it has the right role and the right name?). Same below.
If that doesn't work it means there's an accessibility problem (but sometimes it's not super easy to debug, so feel free to use getByText if you have issues and can't figure them out)